docs/modules/servers/pages/distributed/architecture/specialized-instances.md

```diff
@@ -0,0 +1,37 @@
+= Distributed James Server &mdash; Specialized instances
+:navtitle: Specialized instances
+
+While it is perfectly possible to deploy homogeneous James instances, with the same configuration and thus the sames 
+protocols and the same responsibilities one might want to investigate in 'Specialized instances'.
+
+This deployment topology consist of Distributed James servers with heterogeneous configurations on top of shared 
+data-bases. Groups of James servers will thus handle various protocols and have different responsibilities.
+
+This approach limits cascading failures across protocols and services. Think of *OutOfMemoryErrors*, Cassandra driver 
+queue overuse, CPUs starvation...
+
+However, we can't speak of micro-services here: all James instances runs the same code, James is still a monolith, and
+databases need to be shared across instances.
+
+image::specialized-instances.png[Example of Specialized instances topology]
+
+We speak of :
+
+ - **Front-line servers** serves protocols. James enables to easily turn protocols on and off. Typically, each protocol would 
+ be isolated in its own group of James instances: james-imap, james-jmap, james-smtp, james-webadmin, etc... Refer to
+ protocols configuration files to learn more.
```

Comment:
link?

Reply:
IMO overkill

Reply:
even something like that? [james-imap](http://james.apache.org/server/config-imap4.html) , etc.